src/EA.Weee.RequestHandlers/AatfEvidence/GetEvidenceNoteForSchemeRequest.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace EA.Weee.RequestHandlers.AatfEvidence
+{
+    using System.Threading.Tasks;
+    using Domain.Evidence;
+    using EA.Prsd.Core.Mapper;
+    using EA.Weee.Core.AatfEvidence;
+    using EA.Weee.DataAccess.DataAccess;
+    using Prsd.Core;
+    using Prsd.Core.Mediator;
+    using Requests.AatfEvidence;
+    using Security;
+
+    public class GetEvidenceNoteForSchemeRequest : IRequestHandler<GetEvidenceNoteRequest, EvidenceNoteData>
```

Comment:
Sorry didn't noticed this before but this should be called GetEvidenceNoteForSchemeRequestHandler. It will need a GetEvidenceNoteForSchemeRequest otherwise the mediator will not be able to determine the correct handler. Ie. differentiate between the standard GetEvidenceNoteRequestHandler and the GetEvidenceNoteForSchemeRequestHandler.

Reply:
request has been added and is used by mediator

Reply:
file renamed and request created